compiler/src/iree/compiler/Dialect/LinalgExt/IR/test/roundtrip.mlir

```diff
@@ -592,6 +592,56 @@ func.func @scatter_update_slice_2D(
 
 // -----
 
+func.func @gather_static(
```

Comment:
nit: it looks better to me if we name it as `gather_static_1D`, which tells us what the difference between this test and the below test.

Reply:
I think I misread the test cases. I think the below one is `gather_static_2D_with_batch`, and this can still be `gather_static`.